### PR TITLE
Add script file and Update style

### PR DESCRIPTION
**Pull request by phelian23**
Functionality was added to the menu so that when the user clicks (tap) the icon, the menu items display, and when the user clicks (tap) the menu items, a correct part of the page is displayed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,10 +19,11 @@
             <a class="logo">Femi</a>
             <a class="togglebar" href="#"><img src="Images/Icon-Menu.png" alt="menu icon"></a>
             <ul class="nav-items">
-                <li><a href="#">portfolio</a></li>
-                <li><a href="#">about</a></li>
-                <li><a href="#">contact</a></li>
-                <li><a href="#"><img src="Images/Icon-Mail.png" alt="mail icon"></a></li>
+                <li class="close"><a href="#"><img src="Images/Icon-Cancel.png" alt="Cancel Icon"></a></li>
```
Comment (GzimAsani) [inline index.html:22]: - [ ] This **X**  should not be present in desktop version, please remove it Screenshot from 2021-10-04 20-21-18
 - [ ]

File: index.html [hunk 2]
```diff
@@ -19,10 +19,11 @@
...
+                <li class="show-list"><a href="#portfolio">Portfolio</a></li>
+                <li class="show-list"><a href="#about">About</a></li>
+                <li class="show-list"><a href="#contact">Contact</a></li>
```
Comment (GzimAsani) [inline index.html:25]: - [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

File: index.html [hunk 3]
```diff
@@ -19,10 +19,11 @@
             <a class="logo">Femi</a>
```
Comment (GzimAsani) [inline index.html:19]: - [ ] Your page is overlapping, kindly fix that.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi **Team** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] Add a more descriptive pull request. Explaining what changes did you make
- [ ] _Check the comments under the review for more information._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (phelian23) [reply index.html:19]: Fixed, thanks.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi again **Team**

Good job so far, you implemented 95% of my previous requested changes. But it seems that you've forgotten to implement 1 of them.

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] Add a more descriptive pull request. Explaining what changes did you make(_Pull Request should have a short summary describing introduced changes in more detail._)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @phelian23 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**